posthog/temporal/data_imports/metrics.py

```diff
@@ -0,0 +1,10 @@
+from temporalio import workflow
+from temporalio.common import MetricCounter
+
+
+def get_data_import_finished_metric(source_type: str, status: str) -> MetricCounter:
```

Comment:
Minor nit: could source_type be typed to `ExternalDataSource.Type` and status be typed to `ExternalDataJob.Status`